golang/Chapter04/model/client.go

```diff
@@ -0,0 +1,112 @@
+package model
+
+import (
+	"fmt"
+	"github.com/go-redis/redis/v7"
+	"log"
+	"time"
+)
+
```

Comment:
There are a couple missing functions from early in the chapter, but there are much better solutions for those, and didn't teach anything really, so no worries.

Reply:
I see you noted them in your readme.